standards/SPIFFE_Trust_Domain_and_Bundle.md

```diff
@@ -78,15 +81,32 @@ The `kty` parameter MUST be set, and its behavior follows [Section 4.1][6] of RF
 #### 4.2.2. Public Key Use
 The `use` parameter MUST be set. Its value indicates the type of identity document (or SVID) that it is authoritative for. At the time of this writing, only two SVID types are supported: `x509-svid` and `jwt-svid`. The values are case sensitive. Please see the respective SVID specifications for more information about `use` values. Clients encountering either a missing `use` parameter or an unknown `use` value MUST ignore the entire JWK element.
 
-## 5. Security Considerations
+## 5. SPIFFE Bundle Map
+A SPIFFE Bundle Map is an object containing a collection of SPIFFE Bundles. The bundles it contains are considered authoritative for the corresponding trust domain name, which is also included.
+
+SPIFFE Bundle Maps are designed for use between SPIFFE control plane implementations, between SPIFFE implementations and the workloads they serve, as well as internally (i.e. as an implementation detail of a SPIFFE control plane).
+
+### 5.1. SPIFFE Bundle Map Format
+SPIFFE Bundle Maps are JSON-encoded structures, encapsulating one or more SPIFFE Bundles and some additional metadata. For an example of a complete SPIFFE Bundle Map, please see [Appendix B. SPIFFE Bundle Map Example](#appendix-b-spiffe-bundle-map-example).
+
+#### 5.1.1. Trust Domains
+A static “trust_domains” key MUST be set, and MAY be empty. Its contents are a map of SPIFFE Bundles, keyed by the trust domain name. For more information on what constitutes a valid trust domain name, please see [Section 2][1] of the SPIFFE ID specification.
```

Comment:
I'm thinking if something like "produced_at" with a timestamp value would be very helpful. If delivered asynchronously it'll be very hard to get a notion of freshness of it. A timestamp would go a long way for debugging purposes.

Alternatively, explicitly allowing additional keys to be present could make something like it an implementers choice.

Reply:
Thank you @arndt-s, I agree having a timestamp has some value. We circulated this idea of having a timestamp or some sort of a refresh hint to the map during the Spiffe community meetings. The conclusion we reached that while adding something to show freshness has some value, keeping the Spec simple and leaving this detail up to the implementers would keep the specification simple and hence increase adoption. 

If we add to the spec a required timestamp/freshness field we would need an agreement about how this field would be consumed, when maps expire, how often we need to refresh. And just adding an optional field without further details and why it is important would add complexity to the Spec with minimal benefit. What do you think?

Reply:
Agreed on the focus of keeping it simple to easy use and adoption.

Closing this comment.